Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -472,6 +472,7 @@ def check_new_version(formula, tap_remote_repo, version: nil, url: nil, tag: nil
 
       def check_throttle(formula, new_version)
         throttled_rate = formula.tap.audit_exceptions.dig(:throttled_formulae, formula.name)
+        throttled_rate ||= formula.livecheck.throttle
```

Comment:
I'll probably flip this (since we want to prioritize DSL now):
```suggestion
        throttled_rate = formula.livecheck.throttle
        throttled_rate ||= formula.tap.audit_exceptions.dig(:throttled_formulae, formula.name)
```